library/src/test/java/org/apache/apex/malhar/lib/window/accumulation/PojoInnerJoinTest.java

```diff
@@ -259,4 +259,31 @@ public void PojoInnerJoinTestMultipleKeys()
     Assert.assertEquals("Josh", testOutClass.getUName());
     Assert.assertEquals(12, testOutClass.getAge());
   }
+
+  @Test
+  public void PojoInnerJoinTestSeparateLeftAndRightKeys()
```

Comment:
No need to add a new test... Just modify the existing test to user the new constructor.